content/docs/utilities/z-index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```

Comment:
```suggestion
You can use these utility classes to control the stacking context. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```

Reply:
These utility classes control the stacking order. Adding a `z-{value}` class to your element will change its order in the stack. Remember to add the `pos-rel` class on the element for `z-index` to work.

Reply:
I wasn't sure about explicitly saying `use pos-rel` because the element could be `fixed` or `absolute` too

Reply:
then remove the sentence entirely. 
PlatformUI disclaimer: Platform UI utilities assume a basic knowledge of CSS and the DOM.

Reply:
Roger that